### PR TITLE
OSDOCS-7286 Z-stream RNs for 4.13.8

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.13

Issue:
[OSDOCS-7286](https://issues.redhat.com/browse/OSDOCS-7286)

Link to docs preview:
[Preview](https://63216--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes#ocp-4-13-8)

QE review:
- [ ] QE has approved this change.

Additional information:
Expected to ship live on Aug. 8, 2023.

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc
Comment (skopacz1) [inline release_notes/ocp-4-13-release-notes.adoc:3073]: the two rh-openstack attributes here aren't resolving correctly in the preview, I believe they need to be wrapped in {} brackets rather than colons.

**Review by skopacz1 — COMMENTED** [review]
Left one comment, otherwise looks good! I'm still new to reviewing, so please wait until @adellape approves my review before implementing any feedback. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc
Comment (cbippley) [reply release_notes/ocp-4-13-release-notes.adoc]: Fixed, thank you!

**Review by adellape — APPROVED** [review]
@skopacz1 @cbippley Peer review comments LGTM 👍

**Pull request merged**
